.github/workflows/ci.yml

```
@@ -1,6 +1,8 @@
 name: CI
 on:
-  - push
+  - push:
+      branches:
+        - main
```

```suggestion
        - master
```

@mbauman 

h/t @giordano